docker/proxy/Dockerfile

```diff
@@ -0,0 +1,17 @@
+###############################################
```

Comment:
I actually noticed that nginx has [their own official Docker image](https://hub.docker.com/r/_/nginx/) that should be able to do everything that you're doing here. Any reason not to use it instead?